src/NUnitFramework/tests/Internal/PlatformDetectionTests.cs

```diff
@@ -429,9 +429,9 @@ public void PlatformAttribute_IncludeAndExclude()
         public void PlatformAttribute_InvalidPlatform()
         {
             PlatformAttribute attr = new PlatformAttribute( "Net-1.0,Net11,Mono" );
-            Assert.IsFalse( winXPHelper.IsPlatformSupported( attr ) );
-            Assert.That( winXPHelper.Reason, Does.StartWith("Invalid platform name"));
-            Assert.That( winXPHelper.Reason, Does.Contain("Net11"));
```

Comment:
We seem to have lost this specific assertion on line 434: that the message contains the invalid platform name. Can we add that back in somewhere? It could be here - but actually, I agree with the changes you've made - at this level, we only care that the expected type of exception is being thrown. 

I think I nicer solution might be to add this line to your new 'InvalidPlatformAttributeIsNotRunnable' test - and check that the skipreason contains the invalid platform name. 🙂